lib/google_visualr/base_chart.rb

```diff
@@ -1,22 +1,42 @@
 module GoogleVisualr
 
   class BaseChart
-    include GoogleVisualr::Packages
     include GoogleVisualr::ParamHelpers
 
     DEFAULT_VERSION = "1.0".freeze
 
-    attr_accessor :data_table, :listeners, :version
+    attr_accessor :data_table, :listeners, :version, :material
 
     def initialize(data_table, options={})
-      @data_table = data_table
-      self.version = options.delete(:version) || DEFAULT_VERSION
+      @data_table  = data_table
+      @listeners   = []
+      @version     = options.delete(:version)  || DEFAULT_VERSION
+      @material    = options.delete(:material) || false
       send(:options=, options)
-      @listeners  = []
+    end
+
+    def package_name
+      self.class.to_s.split("::").last.downcase
+    end
+
+    def class_name
+      self.class.to_s.split('::').last
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.